modular_skyrat/code/modules/cargo/packs/bloat.dm

```diff
@@ -21,6 +21,22 @@
 					/obj/item/gun/energy/e_gun/stun)
 	crate_name = "swat taser crate"
 
+/datum/supply_pack/security/armory/swat_tactical
+	name = "SWAT Tactical Operations crate"
+	desc = "Contains two sets of lightweight equipment for tactical operations. Each set contains a vest, night vision helmet, mask, combat belt, and combat gloves. Requires Armory access to open."
+	cost = 6000
+	contains = list(/obj/item/clothing/head/helmet/advanced,
```

Comment:
Wheres the Mark II Techarmor?

Reply:
It's lightweight swat gear, not tech armor

Reply:
The tech armor was included in the node. Include the Mark II Techarmor in this crate aswell.